src/OpenLoco/Map/TileManager.cpp

```diff
@@ -549,6 +549,64 @@ namespace OpenLoco::Map::TileManager
         _numAnimations = 0;
     }
 
+    // 0x004C5596
+    uint16_t countSurroundingWaterTiles(Map::Pos2 pos)
+    {
+        pos.x = (pos.x & 0xFFE0) - 0xA0;
```

Comment:
`- 5 * 32`.

I would convert this at this point to a `TilePos2`. So something like: 

```
// Search a 10x10 area centred at Pos
// Initial tileposition is top left of area
auto tilePos = TilePos(pos) - TilePos(5, 5);
```